Src/Core/CaseStudy.Core/Contracts/IUnitOfWork/IUserFavServices.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CaseStudy.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CaseStudy.Core.Contracts.IUnitOfWork
+{
+    public interface IUserFavServices
+    {
+        public  Task<IEnumerable<Cars>> GetFavCars(int userId);
```

Comment:
no need of access modifiers for methods or properties in interface, as these can be overriden by base class.https://www.tutorialsteacher.com/csharp/csharp-interface#:~:text=Interface%20members%20are%20by%20default,partial%20modifiers%20on%20certain%20conditions.

Reply:
okay